projects/igniteui-angular/src/lib/grids/summaries/summary-row.component.html

```diff
@@ -45,4 +45,25 @@
             [style.flex-basis]="col.getCellWidth()">
         </igx-grid-summary-cell>
     </ng-template>
+    <ng-container *ngIf="pinnedColumns.length > 0 && !grid.isPinningToStart">
```

Comment:
Can't we have this as a template ref so that both left and right summaries can use the same?